Simplenote/UIActivityViewController+Simplenote.swift

```diff
@@ -13,9 +13,10 @@ extension UIActivityViewController {
         guard let content = note.content else {
             return nil
         }
+        let shareFilename = note.exportFilename()
 
         let print = SPSimpleTextPrintFormatter(text: content)
-        let source = SimplenoteActivityItemSource(note: note)
+        let source = SimplenoteActivityItemSource(content: content, filename: String(format: "%@.txt", shareFilename))
```

Comment:
(From the outside, as a reader) it's not clear why an `ActivityItemSource` would need a filename, nor why it would need a `.txt` file extension.

How about if, instead, we pass along the document Unique ID, and deal with filenames / extensions in `SimplenoteActivityItemSource` itself?

```
SimplenoteActivityItemSource(content: content, identifier: note.simperiumKey)
```

(Otherwise the API is "leaking" internal details).